/*
 * Decompiled with CFR 0.152.
 * 
 * Could not load the following classes:
 *  javax.microedition.lcdui.Graphics
 */
import javax.microedition.lcdui.Graphics;

public final class ho
extends ht
implements bq {
    private ox k;
    private fg p;
    private fg q;
    private dc r;
    private ay s;
    private ay t;
    private k u = null;
    private k v = null;
    private lh w;
    private int x;
    private az y;
    private az z;
    private byte A = 0;
    private byte[][] B;
    private ex C;
    private aq[] D;
    private int E;
    private String F;
    private final Object G = new Object();
    private fw H;
    private int I = 0;
    private lm J;
    private int K;
    private long L;
    private String M;
    private boolean N = false;
    private boolean O = false;
    private k P;
    private String Q;
    private by R;
    private d S;

    public ho(String string, String string2, ox ox2, d d2) {
        this.F = string;
        this.M = string2;
        if (d2 == null) {
            d2 = new if(new int[]{0xFF0000, 0xFFFF00});
        }
        this.S = d2;
        this.k = new ox(ox2);
        this.b(241231);
        this.a(new ba());
        this.a(this);
        int n2 = 240;
        int n3 = 320 - ba.a;
        if (com.mg.sq.a.k == 1) {
            n2 = 320;
            n3 = v.u - ba.a;
        }
        int n4 = v.t >= n2 ? (v.t - n2) / 2 : 0;
        int n5 = v.u >= n3 ? (v.u - ba.a - n3) / 2 : 0;
        this.P = new k(n4, n5, n2, n3);
        this.t();
        az az2 = this.y;
        ho ho2 = this;
        ho2.b(az2, true);
        this.a((az)null);
    }

    private void t() {
        int n2 = v.t - 20;
        this.u = new k((v.t - n2) / 2, 140, n2, this.g - 174);
        this.v = new k((v.t - n2) / 2, 30, n2, 44);
        this.p = new fg(false);
        this.p.e(true);
        this.p.i = new k(this.u.a, this.u.b, this.u.c, this.u.d);
        this.w = go.k.a();
        Object object = this.w;
        int n3 = go.l.length - object.D.length;
        int n4 = 0;
        while (n4 < go.m.length) {
            n3 = go.m[n4].l == 1 ? (n3 += go.m[n4].g) : (go.m[n4].l > 1 ? (n3 += go.m[n4].g / go.m[n4].l + (go.m[n4].g % go.m[n4].l > 0 ? 1 : 0)) : ++n3);
            ++n4;
        }
        this.x = n3;
        if (this.x < go.n) {
            this.x = go.n;
        }
        this.p.d(this.x, 1);
        this.p.a(this);
        ll[] llArray = go.l;
        object = this;
        n4 = object.w.D.length;
        object.p.t();
        int n5 = 0;
        while (n5 < llArray.length) {
            if (llArray[n5] != null) {
                boolean bl2 = false;
                if (n4 > 0) {
                    int n6 = 0;
                    while (n6 < object.w.D.length) {
                        if (llArray[n5].c.equals(object.w.D[n6].c)) {
                            --n4;
                            bl2 = true;
                            break;
                        }
                        ++n6;
                    }
                }
                if (!bl2) {
                    dc dc2 = new dc(object.k.a(mb.a(llArray[n5]) + 98, true), llArray[n5], 0, object.S);
                    object.p.a((Object)dc2);
                }
            }
            ++n5;
        }
        this.q = new fg(false);
        this.q.i = new k(this.v.a, this.v.b, this.v.c, this.v.d);
        this.q.d(6, 0);
        this.q.a(this);
        this.q.j = true;
        this.q.d(true);
        this.q.i(0);
        object = new lm[go.m.length];
        int n7 = 0;
        while (n7 < ((lm[])object).length) {
            object[n7] = go.m[n7].b();
            ++n7;
        }
        this.a((lm[])object);
        object = this;
        this.B = new byte[4][3];
        object.B[0] = new byte[]{1, -1, 1, -1};
        byte[] byArray = new byte[4];
        byArray[0] = 2;
        byArray[2] = 2;
        object.B[1] = byArray;
        object.B[2] = new byte[]{-1, 1, -1, 1};
        this.t = new ay();
        this.t.a(new k(this.v.a, this.v.b, this.v.c, this.v.d + 2));
        this.t.b(this.q);
        this.t.h(2);
        this.s = new ay();
        this.s.a(new k(this.u.a, this.u.b, this.u.c, this.u.d));
        this.s.b(this.p);
        this.s.h(2);
        this.C = new ex("K\u1ebft h\u1ee3p", 2);
        n7 = bx.d.a("K\u1ebft h\u1ee3p") + 10;
        this.C.a(this.c + (v.t - n7) / 2, this.u.b - 25, n7, 18);
        this.D = new aq[]{this.t, this.C, this.s};
        this.R = new by(0xFF0000);
        this.y = new gb(1, 3);
        this.z = new gb(3, 2);
    }

    private void a(lm[] lmArray) {
        int n2 = 0;
        while (n2 < lmArray.length) {
            int n3;
            int n4 = n3 = lmArray[n2].e == 3 ? 2 : 1;
            if (lmArray[n2].g > 0) {
                if (lmArray[n2].l > 0) {
                    int n5 = lmArray[n2].g;
                    int n6 = lmArray[n2].g / lmArray[n2].l;
                    int n7 = 0;
                    while (n7 < n6) {
                        lm lm2 = lmArray[n2].b();
                        lmArray[n2].b().g = lmArray[n2].l;
                        n5 -= lmArray[n2].l;
                        this.p.a((Object)new dc(null, lm2, n3, this.S));
                        ++n7;
                    }
                    if (n5 > 0) {
                        lm lm3 = lmArray[n2].b();
                        lmArray[n2].b().g = n5;
                        this.p.a((Object)new dc(null, lm3, n3, this.S));
                    }
                } else {
                    this.p.a((Object)new dc(null, lmArray[n2], n3, this.S));
                }
            }
            ++n2;
        }
    }

    /*
     * WARNING - Removed try catching itself - possible behaviour change.
     */
    private void v() {
        Object object = this.G;
        synchronized (object) {
            dc dc2;
            this.e(true);
            int n2 = 0;
            int n3 = 0;
            int n4 = 0;
            while (n4 < this.q.s()) {
                dc2 = (dc)this.q.k(n4);
                if (dc2.j == 0) {
                    if (dc2.k != null) {
                        ++n2;
                    }
                } else if (dc2.j == 1) {
                    ++n3;
                }
                ++n4;
            }
            n4 = 0;
            int n5 = 0;
            String[] stringArray = new String[n2];
            int[] nArray = new int[n3];
            int[] nArray2 = new int[n3];
            int n6 = 0;
            while (n6 < this.q.s()) {
                dc2 = (dc)this.q.k(n6);
                if (dc2.j == 0) {
                    if (dc2.k != null) {
                        stringArray[n4] = ((ll)dc2.k).c;
                        ++n4;
                    }
                } else if (dc2.j == 1 && dc2.k != null) {
                    nArray[n5] = ((lm)dc2.k).a;
                    nArray2[n5] = ((lm)dc2.k).g;
                    ++n5;
                }
                ++n6;
            }
            ks.a().a(this.F, stringArray, nArray, nArray2, this.L);
            com.mg.sq.a.s().a((String)null, (il)null);
            return;
        }
    }

    protected final boolean g(int n2) {
        switch (n2) {
            case 4: {
                if (this.r.k == null) break;
                ho ho2 = this;
                if (ho2.r.k != null) {
                    if (ho2.r.j == 0) {
                        ks.a().a(ho2.F, (byte)1, ((ll)ho2.r.k).c);
                    } else {
                        ho2.J = (lm)ho2.r.k;
                        int n3 = 0;
                        int n4 = 0;
                        while (n4 < ho2.q.s()) {
                            lm lm2;
                            if ((((dc)ho2.q.k((int)n4)).j == 1 || ((dc)ho2.q.k((int)n4)).j == 2) && (lm2 = (lm)((dc)ho2.q.k((int)n4)).k) != null && lm2.a == ho2.J.a) {
                                n3 += lm2.g;
                            }
                            ++n4;
                        }
                        ks.a().a(ho2.F, (byte)1, ho2.J.a, n3);
                    }
                }
                ho2.N = true;
                com.mg.sq.a.s().a((String)null, (il)null);
                if (this.m == null) break;
                this.A();
                break;
            }
            case 5: {
                gu gu2;
                if (!this.l.c(-7524) || (gu2 = (gu)this.l.d(-7524)) == null) break;
                this.J = gu2.t();
                int n5 = this.J.g = gu2.u();
                ks.a().a(this.F, (byte)0, this.J.a, n5);
                com.mg.sq.a.s().a((String)null, (il)null);
                this.l.a(-7524, false);
                break;
            }
            case 6: {
                this.l.a(false);
                break;
            }
            case 2: {
                if (this.K == 0 && !this.O) {
                    ho ho3 = this;
                    al al2 = ho3.l.a("Ch\u00fa \u00fd", "Ch\u01b0a \u0111\u1ee7 nguy\u00ean li\u1ec7u. Vui l\u00f2ng th\u1eed l\u1ea1i!!!", "\u0110\u00f3ng", 1, 1);
                    al2.b(199199);
                    al2.a(ho3);
                    ho3.l.a(al2, false);
                    break;
                }
                if (this.K != 1 || this.O) break;
                if (go.s > -1L && this.L > go.s) {
                    ho ho4 = this;
                    al al3 = ho4.l.a("Ch\u00fa \u00fd", "V\u01b0\u1ee3t qu\u00e1 s\u1ed1 ti\u1ec1n b\u1ea1n \u0111ang c\u00f3. Vui l\u00f2ng th\u1eed l\u1ea1i!!!", "\u0110\u00f3ng", 1, 1);
                    al3.b(199199);
                    al3.a(ho4);
                    ho4.l.a(al3, false);
                    break;
                }
                ho ho5 = this;
                hc hc2 = new hc(String.valueOf(ho5.M) + ". B\u1ea1n c\u00f3 mu\u1ed1n k\u1ebft h\u1ee3p kh\u00f4ng?", "K\u1ebft h\u1ee3p", 7, "Kh\u00f4ng", 1);
                hc2.a(ho5);
                ho5.l.a(hc2, false);
                break;
            }
            case 7: {
                this.l.a(false);
                this.v();
            }
        }
        return false;
    }

    /*
     * WARNING - Removed try catching itself - possible behaviour change.
     */
    public final void a(ll[] llArray, lm[] lmArray, byte by2) {
        this.l.v();
        Object object = this.G;
        synchronized (object) {
            dc dc2;
            this.e(true);
            int n2 = 0;
            while (n2 < this.q.s()) {
                dc2 = (dc)this.q.k(n2);
                if (dc2.k != null) {
                    if (dc2.j == 0) {
                        go.b((ll)dc2.k);
                    } else {
                        go.a((lm)dc2.k);
                    }
                }
                ++n2;
            }
            this.q.t();
            n2 = 0;
            while (n2 < llArray.length) {
                if (llArray[n2] != null) {
                    dc2 = new dc(this.k.a(mb.a(llArray[n2]) + 98, true), llArray[n2], 0, this.S);
                    this.q.a((Object)dc2);
                    go.a(llArray[n2]);
                }
                ++n2;
            }
            n2 = 0;
            while (n2 < lmArray.length) {
                if (lmArray[n2] != null) {
                    dc2 = new dc(null, lmArray[n2], lmArray[n2].e, this.S);
                    this.q.a((Object)dc2);
                    go.a(lmArray[n2], lmArray[n2].g);
                }
                ++n2;
            }
            this.Q = by2 == 1 ? "K\u1ebft h\u1ee3p th\u00e0nh c\u00f4ng" : "K\u1ebft h\u1ee3p th\u1ea5t b\u1ea1i";
            this.O = true;
            return;
        }
    }

    protected final void e(int n2) {
        switch (n2) {
            case 1: {
                if (this.m != null) {
                    this.A();
                    return;
                }
                this.l.a(false);
                this.l.M();
                return;
            }
            case 3: {
                this.c(95);
            }
        }
    }

    protected final void f(int n2) {
        byte by2 = this.A;
        switch (n2) {
            case 96: 
            case 97: 
            case 98: 
            case 99: {
                if (!this.D[this.A].f(n2)) {
                    int n3 = n2 - 96;
                    ho ho2 = this;
                    if (n3 >= 0 && (n3 = ho2.B[ho2.A][n3]) >= 0) {
                        ho2.A = (byte)n3;
                    }
                }
                if (by2 != this.A) {
                    this.D[this.A].d(true);
                    this.D[by2].d(false);
                    if (this.D[this.A] instanceof ay) {
                        fg fg2 = (fg)((ay)this.D[this.A]).w();
                        fg2.i(0);
                    }
                }
                if (this.p.m()) {
                    this.a(this.p.r());
                    return;
                }
                if (this.q.m()) {
                    this.a(this.q.r());
                    return;
                }
                this.H = null;
                return;
            }
            case 95: {
                if (this.D[1].m()) {
                    this.i.d(0, ((ex)this.D[1]).a());
                    return;
                }
                this.D[this.A].f(n2);
                return;
            }
        }
        this.D[this.A].f(n2);
    }

    public final void f(int n2, int n3) {
        this.e(true);
        if (this.m != null) {
            boolean bl2;
            block10: {
                int n4 = n3;
                int n5 = n2;
                az[] azArray = this.m.a();
                ho ho2 = this;
                int n6 = 0;
                while (n6 < azArray.length) {
                    if (azArray[n6] != null && azArray[n6].a(n5, n4)) {
                        if (ho2.i != null) {
                            ho2.i.d(-1, azArray[n6].a());
                        }
                        bl2 = true;
                        break block10;
                    }
                    ++n6;
                }
                bl2 = false;
            }
            if (bl2) {
                return;
            }
            if (this.m.c(n2, n3)) {
                return;
            }
            this.A();
            return;
        }
        byte by2 = this.A;
        int n7 = 0;
        while (n7 < this.D.length) {
            k k2 = this.D[n7].equals(this.s) ? new k(this.p.i.a + this.P.a, this.p.i.b + this.P.b, this.p.i.c, this.p.i.d) : (this.D[n7].equals(this.t) ? new k(this.q.i.a + this.P.a, this.q.i.b + this.P.b, this.q.i.c, this.q.i.d) : new k(this.D[n7].c() + this.P.a, this.D[n7].d() + this.P.b, this.D[n7].e(), this.D[n7].f()));
            if (k2.a(n2, n3)) {
                if (n7 != by2) {
                    this.A = (byte)n7;
                    this.D[by2].d(false);
                    this.D[this.A].d(true);
                }
                this.D[n7].c(n2 - this.P.a, n3 - this.P.b);
                if (this.D[n7] instanceof ex) {
                    this.d(-1, ((ex)this.D[n7]).a());
                }
                return;
            }
            n7 = (byte)(n7 + 1);
        }
    }

    public final void e(int n2, int n3) {
        if (this.p.m()) {
            this.p.e(n2, n3);
        }
    }

    private void a(Object object) {
        this.I = 0;
        this.H = null;
        this.r = (dc)object;
    }

    public final void u() {
        int n2 = 0;
        while (n2 < this.D.length) {
            if (this.D[n2] != null) {
                this.D[n2].n();
                this.D[n2].c(true);
            }
            ++n2;
        }
        if (this.H != null) {
            this.H.i();
        }
        if (this.I < 7) {
            ++this.I;
            if (this.I == 7 && this.r != null && this.r.k != null) {
                Object object = this.r.k;
                ho ho2 = this;
                this.H = new fw(object);
                int n3 = ho2.g - 18;
                ho2.H.a(9, n3, ho2.f - 20, ho2.g / 4);
                ho2.H.c(9, n3);
                ho2.H.a(9, n3 - ho2.H.q() - 7);
            }
        }
        if (this.h) {
            --this.E;
            if (this.E <= 0) {
                this.e(false);
                this.E = 0;
            }
        }
    }

    /*
     * WARNING - Removed try catching itself - possible behaviour change.
     */
    public final void a(String string, String object, byte by2, long l2) {
        this.l.v();
        Object object2 = this.G;
        synchronized (object2) {
            this.e(true);
            this.K = by2;
            this.M = object;
            this.L = l2;
            int n2 = 0;
            while (n2 < this.q.s()) {
                object = (dc)this.q.k(n2);
                if (((dc)object).j == 0 && ((dc)object).k != null) {
                    ll ll2 = (ll)((dc)object).k;
                    if (ll2.c.equals(string)) {
                        this.q.a((dc)object);
                        this.p.a(object);
                        this.a(this.q.r());
                        return;
                    }
                }
                ++n2;
            }
            return;
        }
    }

    public final void a(String object, byte by2, long l2) {
        this.l.v();
        if (this.N) {
            this.K = by2;
            this.M = object;
            this.L = l2;
            this.q.a(this.r);
            lm lm2 = (lm)this.r.k;
            object = this;
            int n2 = 0;
            a a2 = new a();
            int n3 = 0;
            while (n3 < ((ho)object).p.s()) {
                lm lm3;
                if ((((dc)((ho)object).p.k((int)n3)).j == 1 || ((dc)((ho)object).p.k((int)n3)).j == 2) && (lm3 = (lm)((dc)((ho)object).p.k((int)n3)).k) != null && lm3.a == lm2.a) {
                    n2 += lm3.g;
                    a2.a(((ho)object).p.k(n3));
                }
                ++n3;
            }
            n2 += lm2.g;
            n3 = 0;
            while (n3 < a2.d()) {
                ((ho)object).p.a((dc)a2.b(n3));
                ++n3;
            }
            if (lm2.l <= 0) {
                lm lm4 = lm2.b();
                lm2.b().g = n2;
                dc dc2 = new dc(null, lm4, lm2.e == 3 ? 2 : 1, ((ho)object).S);
                ((ho)object).p.a((Object)dc2);
            } else {
                n3 = n2 / lm2.l + (n2 % lm2.l > 0 ? 1 : 0);
                int n4 = 0;
                while (n4 < n3) {
                    Object object2 = lm2.b();
                    if (n2 >= lm2.l) {
                        ((lm)object2).g = lm2.l;
                        n2 -= lm2.l;
                    } else if (n2 > 0) {
                        ((lm)object2).g = n2;
                        n2 = 0;
                    }
                    object2 = new dc(null, object2, lm2.e == 3 ? 2 : 1, ((ho)object).S);
                    ((ho)object).p.a(object2);
                    ++n4;
                }
            }
            this.w();
            this.N = false;
            this.r = null;
            this.a(this.q.r());
            return;
        }
    }

    /*
     * WARNING - Removed try catching itself - possible behaviour change.
     */
    public final void b(String string, String object, byte by2, long l2) {
        this.l.v();
        Object object2 = this.G;
        synchronized (object2) {
            this.e(true);
            this.K = by2;
            this.M = object;
            this.L = l2;
            int n2 = 0;
            while (n2 < this.p.s()) {
                object = (dc)this.p.k(n2);
                if (((dc)object).j == 0 && ((dc)object).k != null) {
                    ll ll2 = (ll)((dc)object).k;
                    if (ll2.c.equals(string)) {
                        this.p.a((dc)object);
                        this.q.a(object);
                        this.a(this.p.r());
                        return;
                    }
                }
                ++n2;
            }
            return;
        }
    }

    public final void b(String object, byte by2, long l2) {
        this.l.v();
        if (this.J != null) {
            this.K = by2;
            this.M = object;
            this.L = l2;
            by2 = 0;
            a a2 = new a();
            int n2 = 0;
            while (n2 < this.q.s()) {
                if (((dc)this.q.k((int)n2)).j == 1 && (object = (lm)((dc)this.q.k((int)n2)).k) != null && ((ld)object).a == this.J.a) {
                    by2 = (byte)(by2 + ((lm)object).g);
                    a2.a(this.q.k(n2));
                }
                ++n2;
            }
            by2 = (byte)(by2 + this.J.g);
            n2 = 0;
            while (n2 < a2.d()) {
                this.q.a((dc)a2.b(n2));
                ++n2;
            }
            if (this.J.l <= 0) {
                object = this.J.b();
                this.J.b().g = by2;
                dc dc2 = new dc(null, object, 1, this.S);
                this.q.a((Object)dc2);
            } else {
                n2 = by2 / this.J.l + (by2 % this.J.l > 0 ? 1 : 0);
                int n3 = 0;
                while (n3 < n2) {
                    object = this.J.b();
                    if (by2 >= this.J.l) {
                        ((lm)object).g = this.J.l;
                        by2 = (byte)(by2 - this.J.l);
                    } else if (by2 > 0) {
                        ((lm)object).g = by2;
                        by2 = 0;
                    }
                    object = new dc(null, object, 1, this.S);
                    this.q.a(object);
                    ++n3;
                }
            }
            lm lm2 = this.J;
            object = this;
            int n4 = 0;
            lm lm3 = lm2.b();
            a a3 = new a();
            int n5 = 0;
            while (n5 < ((ho)object).p.s()) {
                lm lm4;
                ((ho)object).r = (dc)((ho)object).p.k(n5);
                if (((ho)object).r != null && ((ho)object).r.k != null && (((ho)object).r.j == 1 || ((ho)object).r.j == 2) && (lm4 = (lm)((ho)object).r.k) != null && lm4.a == lm2.a) {
                    n4 += lm4.g;
                    a3.a(((ho)object).r);
                }
                ++n5;
            }
            lm3.g = n4 - lm2.g;
            n5 = 0;
            while (n5 < a3.d()) {
                ((ho)object).p.a((dc)a3.b(n5));
                ++n5;
            }
            if (lm3.g > 0) {
                super.a(new lm[]{lm3});
            }
            super.w();
            this.a(this.p.r());
        }
    }

    public final void e(boolean bl2) {
        super.e(bl2);
        this.E = 5;
    }

    public final void b(Graphics graphics) {
        this.R.a(graphics, "Ph\u00ed k\u1ebft h\u1ee3p: " + i.a(this.L, ",") + " KEN", (v.t - bx.d.a("Ph\u00ed k\u1ebft h\u1ee3p: ")) / 2 - 10, 10, 0);
        int n2 = 0;
        if (!this.O) {
            if (this.M != null) {
                String[] stringArray = bx.a(this.M, v.t - 20);
                int n3 = 0;
                while (n3 < stringArray.length) {
                    this.R.a(graphics, stringArray[n3], (v.t - bx.d.a(stringArray[n3])) / 2, this.q.d() + this.q.f() + 8 + n2, 0);
                    n2 += this.R.a();
                    ++n3;
                }
            }
        } else if (this.Q != null) {
            this.R.a(graphics, this.Q, (v.t - bx.d.a(this.Q)) / 2, this.q.d() + this.q.f() + 8, 0);
        }
        if (this.t != null) {
            this.t.a(graphics, this.c, this.d);
            this.t.c(true);
        }
        this.C.a(graphics, 0, 12);
        if (this.s != null) {
            this.s.a(graphics, this.c, this.d + 12);
            this.s.c(true);
        }
        if (!this.n && this.H != null) {
            this.H.a(graphics, this.c, this.d);
        }
        bx.d.a(graphics, String.valueOf(this.p.s()) + "/" + go.n, this.u.a, this.u.b - 16, 0);
    }

    private void w() {
        this.p.i(this.p.q());
    }

    public final void a(Graphics graphics) {
        pc.a(graphics, this.c, this.d, this.f, this.g, v.aj, true);
    }

    public final void b(aq object, int n2) {
        this.a(object);
        if (this.D[this.A] instanceof ay) {
            k k2 = ((fg)((ay)this.D[this.A]).w()).u();
            object = this;
            if (((ho)object).r != null) {
                Object object2;
                bs bs2 = new bs();
                if (((ho)object).t.m() && !((ho)object).O && ((ho)object).q != null && ((ho)object).r.j != 3) {
                    bs2.a(new br("B\u1ecf ra", 4));
                }
                if (((ho)object).s.m() && !((ho)object).O && ((ho)object).p != null) {
                    Object object3 = object;
                    int n3 = 0;
                    int n4 = 0;
                    while (n4 < ((ho)object3).q.a()) {
                        if (((ho)object3).q.k(n4) != null) {
                            ++n3;
                        }
                        ++n4;
                    }
                    if (n3 < ((ho)object).q.a()) {
                        object2 = object3 = object;
                        if (((ho)object3).r.k != null) {
                            object2 = object3;
                            if (((ho)object2).r.j == 1) {
                                object2 = object3;
                                Object object4 = (lm)((ho)object2).r.k;
                                object2 = object3;
                                Object object5 = new dc(null, ((lm)object4).b(), ((ho)object2).r.j, ((ho)object3).S);
                                object5 = new gu((dc)object5);
                                ((am)object5).a((bf)object3);
                                ((gu)object5).e(((lm)object4).g);
                                object4 = new bd("Xong", 5);
                                object2 = object5;
                                ((am)object2).a((az)object4, true);
                                ((am)object5).a(new bd("", 5));
                                object4 = new bd("H\u1ee7y", 6);
                                object2 = object5;
                                ((am)object2).b((az)object4, true);
                                ((am)object5).b(-7524);
                                ((gu)object5).j(true);
                                ((ht)object3).l.a((al)object5, false);
                            } else {
                                object2 = object3;
                                if (((ho)object2).r.j == 0) {
                                    object2 = object3;
                                    ks.a().a(((ho)object3).F, (byte)0, ((ll)((ho)object2).r.k).c);
                                    com.mg.sq.a.s().a((String)null, (il)null);
                                }
                            }
                        }
                    }
                }
                if (bs2.s() == null || bs2.s().length == 0) {
                    return;
                }
                int n5 = k2.a + ((al)object).c + (k2.c - bs2.e()) / 2;
                int n6 = k2.b + ((al)object).d + k2.d;
                if (n6 + bs2.f() > v.u - ba.a) {
                    n6 = v.u - ba.a - bs2.f();
                }
                bs2.a_(((al)object).c + ((am)object).f + bs2.e(), n6);
                bs2.d(n5 < ((al)object).c ? ((al)object).c : (n5 + bs2.e() > ((al)object).c + ((am)object).f ? ((al)object).c + ((am)object).f - bs2.e() : n5), n6);
                bs2.a((bf)object);
                az az2 = ((ho)object).y;
                object2 = object;
                ((am)object2).b(az2, true);
                az2 = ((ho)object).z;
                object2 = object;
                ((am)object2).a(az2, true);
                bs2.a_(1);
                ((ht)object).n = true;
                ((ht)object).a(bs2, ((ho)object).z, null, ((ho)object).y);
            }
        }
    }

    public final void a(aq aq2, int n2) {
    }

    public final void a(aq aq2, int n2, int n3) {
    }
}

